public/static/locales/en/donate.json

```diff
@@ -5,7 +5,7 @@
   "yourTreesPlantedByOnLocation_plural": "Your {{treeCount}} trees will be planted by {{projectName}} in {{location}}.",
   "myTreesPlantedByOnLocation": "My {{treeCount}} tree is being planted in {{location}}.",
   "myTreesPlantedByOnLocation_plural": "My {{treeCount}} trees are being planted in {{location}}.",
-  "PROJECTS": "PROJECTS",
```

Comment:
similar changes for `locales/cs/donate.json` has not been made. Why is that?

Reply:
Only English is handled via the code here, the rest is via LingoHub. If something in other languages is missing, or we need to make this change as well, we can do that in a subsequent release.